fw/http_frame.c

```diff
@@ -1878,9 +1878,11 @@ tfw_h2_frame_process(TfwConn *c, struct sk_buff *skb)
 		h2->data_off = 0;
 		h2->skb_head = data_up.skb->next = data_up.skb->prev = NULL;
 		r = tfw_http_msg_process_generic(c, h2->cur_stream, data_up.skb);
-		if (r == T_DROP) {
+		if (r && r != T_POSTPONE) {
 			kfree_skb(nskb);
 			goto out;
+		} else if (r == T_POSTPONE) {
```

Comment:
Please pay attention to this code !!!

Reply:
Determining of postponing occurs in `tfw_h2_frame_recv`. `tfw_http_msg_process_generic` just process current frame and can't return `T_POSTPONED`. In `tfw_http_msg_process_generic`  I can't see any codepath that returns `T_POSTPONED`. Even if `tfw_http_msg_process_generic` could return `T_POSTPONED` it would be senseless, because we already determined postponing above. Checking of `T_POSTPONE` here doesn't have a sense.

Reply:
Can't this path `tfw_http_msg_process_generic()` -> `tfw_http_req_process()` -> `ss_skb_process()` -> `tfw_http_parse_req()` lead to `T_POSTPONED`  ?